src/PreferencesController.ts

```diff
@@ -94,11 +132,67 @@ export class PreferencesController extends BaseController<BaseConfig, Preference
 		return newTokens;
 	}
 
+	/**
+	 * Adds a collectible to the stored collectible list
+	 *
+	 * @param address - Hex address of the collectible contract
+	 * @param tokenId - The NFT identifier
+	 * @returns - Current collectible list
```

Comment:
This should probably say something like `@returns - Promise resolving to the current collectible list` since it's `async`.